### PR TITLE
New: Open Collective post

### PR DESCRIPTION
**Pull request by nzakas**
The blog post announcing our Open Collective page. :fire:

### CODE REVIEW

File: _posts/2019-02-12-funding-eslint-future.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+All of this means that ESLint has become an important part of the JavaScript ecosystem. It's a tool that the JavaScript community depends on and expects to be available, performant, and secure. These expectations are currently being met by an all-volunteer team who spends their free time during lunch breaks, nights, and weekends maintaining ESLint. And while that has worked well for normal maintenance, it's grown difficult to deal with more involved issues such as roadmap planning and security issues.
```
[the remark this answers is not in the document]
Comment (nzakas) [reply _posts/2019-02-12-funding-eslint-future.md]: In this context, "all of this" is correct. https://english.stackexchange.com/questions/77514/all-of-this-vs-all-of-these

File: _posts/2019-02-12-funding-eslint-future.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Today, we are happy to announce the [ESLint Collective](https://opencollective.com/eslint) on Open Collective. Open Collective is a secure way for people to sustain and support open source projects. The funds taken in and paid out are publicly available on each collective's page, so there is 100% transparency around the money a collective uses. Open Collective has already helped other JavaScript ecosystem projects such as [Webpack](https://opencollective.com/webpack) and [Babel](https://opencollective.com/babel) to fund their ongoing development, and the ESLint team is happy to join them on what has proven to be a great option for open source sustainability.
+
+Some of the benefits of working with Open Collective include:
+
+* **Full Transparency.** Everyone gets to see where money is coming from and where it's going.
+* **Individual and Corporate Sponsors.** Open Collective makes it easy to for both individuals and companies to sponsor open source projects.
```
[the remark this answers is not in the document]
Comment (nzakas) [reply _posts/2019-02-12-funding-eslint-future.md]: Yeah, that's a typo.

```suggestion
* **Individual and Corporate Sponsors.** Open Collective makes it easy for both individuals and companies to sponsor open source projects.
```

File: _posts/2019-02-12-funding-eslint-future.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* **Rewarding contributions from the community.** We'd like to reward everyone who contributes to ESLint, whether it's a one-time contribution or ongoing contributions. We are still figuring out the details, but rewarding people with everything from stickers and t-shirts to cash payouts for significant contributions are things we are thinking about right now.
+* **Improve documentation.** ESLint's documentation hasn't had a significant overhaul since the project was first released. We feel there are a lot of ways to improve the documentation, including potentially contracting an information architect or technical writer to improve our documentation.
+* **Better communication and support.* By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
```
Comment (btmills) [inline _posts/2019-02-12-funding-eslint-future.md]:
```suggestion
* **Better communication and support.** By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
```

File: _posts/2019-02-12-funding-eslint-future.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    <a href="https://frontendmasters.com"><img src="https://static.frontendmasters.com/assets/fm/js/frontendmasters.0e71088726.svg" alt="Frontend Masters" height="32"></a>
+</p>
+
+We're also excited to announce our first sponsors. [Facebook](https://facebook.com) and [Airbnb](https://airbnb.com) have both pledged $1,000 per month to support ESLint.
+
+Additionally, [Frontend Masters](https://frontendmasters.com) has pledged $200 per month to support ESLint.
+
+If your company also uses ESLint to find and fix problems in your JavaScript code, please ask them to join these fantastic companies in [sponsoring ESLint](https://opencollective.com/eslint). (All sponsors above $200 per month get their logos on both the ESLint README and on the homepage of eslint.org.)
+
+We'd like to thank Eric Nakagawa at Facebook, Jordan Harband at Airbnb, Marc Grabanski at Frontend Masters, and Pia Mancini at Open Collective for supporting our early sponsorship efforts at their respective companies.
```
Comment (btmills) [inline _posts/2019-02-12-funding-eslint-future.md]: What do you think about linking Twitter or GitHub profiles for these people?

File: _posts/2019-02-12-funding-eslint-future.md (hunk 3 above)
Comment (platinumazure) [inline _posts/2019-02-12-funding-eslint-future.md]: "meetups, companies, and conferences": Two of these are events, one (companies) is a group of entities/locations. This feels like a small parallelism issue, but I'm not sure how best to resolve it (or, for that matter, if it even needs to be resolved).

Also, formatting typo:

```suggestion
* **Better communication and support.** By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
```

Comment (nzakas) [reply _posts/2019-02-12-funding-eslint-future.md]: Yeah, I don't think it's important in this context. All of them are places we couldn't afford to go previously.

File: _posts/2019-02-12-funding-eslint-future.md (hunk 4 above)
Comment (nzakas) [reply _posts/2019-02-12-funding-eslint-future.md]: I don't think its necessary. All of them were working to represent their respective companies, and we already are linking directly to those companies.

File: _posts/2019-02-12-funding-eslint-future.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+<p class="text-center">
+    <a href="https://facebook.com"><img src="/img/logos/facebook.png" alt="Facebook" height="96"></a>
+    <a href="https://airbnb.com"><img src="/img/logos/airbnb.png" alt="Airbnb" height="96"></a><br>
+    <a href="https://frontendmasters.com"><img src="https://static.frontendmasters.com/assets/fm/js/frontendmasters.0e71088726.svg" alt="Frontend Masters" height="32"></a>
+</p>
+
+We're also excited to announce our first sponsors. [Facebook](https://facebook.com) and [Airbnb](https://airbnb.com) have both pledged $1,000 per month to support ESLint.
+
+Additionally, [Frontend Masters](https://frontendmasters.com) has pledged $200 per month to support ESLint.
```
Comment (nzakas) [inline _posts/2019-02-12-funding-eslint-future.md]: I need to double check on this. I thought this was a monthly donation, but it looks like it was a one-time donation. I'm going to try to reach Marc to make sure that was intended so I can reword this.

Comment (nzakas) [reply _posts/2019-02-12-funding-eslint-future.md]: Okay, this was a one-time donation to get us started, I'll update the post.

**Pull request merged**